static/js/message_fetch.js

```diff
@@ -27,6 +27,7 @@ const consts = {
     num_before_home_anchor: 200,
     num_after_home_anchor: 200,
     recent_topics_initial_fetch_size: 400,
+    sent_by_user_fetch_size: 1000,
```

Comment:
We could add a setting which user can configure like:

Expire participation in a topic if no messages are sent by me in  `X days `

Then, we only fetch messages sent by user in the last X days.